tools/clang/test/HLSLFileCheck/hlsl/workgraph/case070_noinput.hlsl

```diff
@@ -20,6 +20,12 @@ void node070_coalescing() { }
 [NumThreads(1,1,1)]
 void node070_thread() { }
 
+[Shader("node")]
+[NodeLaunch("mesh")]
+[NumThreads(1,1,1)]
```

Comment:
Any limit on number threads for mesh node?

Reply:
The same limits as on regular mesh shaders, which is that the maximum threads must be less than 128. As for lower limits, this is fine.

Reply:
Yes, we should limit these to what's allowed for mesh shaders.

Reply:
I saw that as being a part of the second task in #6404. Do you think it needs doing now?

Reply:
I've mentioned this limit in the mesh nodes attribute issue #6443